conf/config.yaml

```diff
@@ -53,7 +53,7 @@ nginx_config:                     # config for render the template to genarate n
 etcd:
   host: "http://127.0.0.1:2379"   # etcd address
   prefix: "/apisix"               # apisix configurations prefix
-  timeout: 1                      # 1 seconds
+  timeout: 3                      # 3 seconds
```

Comment:
Is this a related modification?

Reply:
the travis is slow now, we have to use a longer timeout.Otherwise the test case will randomly fail.

Reply:
I think you should add issue for it, this method will affect all user, not only test cases.